eval/create_eval.sh

```diff
@@ -90,6 +94,11 @@ read -p "Enter the memory in GB (default: 100): " MEMORY
 CPUS=${CPUS:-8}
 MEMORY=${MEMORY:-100}
 
+#
+# Add option to set if the model is deepseek r1
+#
+read -p "Is the model Deepseek R1 (to correctly parse <think> tags)? (y/N): " IS_DEEPSEEK_R1
```

Comment:
Looking how this comment eventually transforms to a `parse_thinking_output` flag (which could in principle apply to other models and also later versions of DeepSeek), this could be renamed to something to the effect of `HAS_THINK_TAG`, or `HAS_THINKING_OUTPUT`, etc.

Reply:
Unfortunately DeepSeek-R1 also uses a custom end-of-token in vllm that we haven't seen with other local models. This bash variable switches on `--inf_is_deepseek_r1`, which not only switches on `parse_thinking_output` but also gets passed to select the custom token in vllm.

https://github.com/kilian-group/phantom-wiki/blob/233-refactor-llm/src/phantom_eval/llm/vllm.py#L46-50

I initially thought of just creating `--inf_parse_thinking_output` flag. But because vllm also checks if the model is deepseek-r1, I just created a more explicit flag in the parser.

I agree this is a hack and I don't expect it to scale as we add more models. I think it's ok for now, as we don't know how vllm/huggingface will handle inference on future models.

Reply:
Okay then for clarity I would add the comment about this additional behavior as well: 

```suggestion
read -p "Is the model DeepSeek R1 (to correctly parse <think> tags and set DeepSeek R1-specific stop tokens)? (y/N): " IS_DEEPSEEK_R1
```